discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/QueryAggregation.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+/**
+ * An aggregation produced by the Discovery service to analyze the input provided.
+ */
+public class QueryAggregation extends GenericModel {
```

Comment:
what about `private String path;` for `nested` aggregation? or `private Long count;` for `term` aggregation or `hits` for `top_hits` aggregation?